fuse_examples/imaging/segmentation/picai/runner.py

```diff
@@ -0,0 +1,381 @@
+
+    #### Train Data
+    # split to folds randomly - temp
+    dataset_all = PICAI.dataset(
+        paths=paths,
+        cfg=train,
+        reset_cache=False,
+        train=True,
+        run_sample=train["run_sample"],
+    )
+    folds = dataset_balanced_division_to_folds(
+        dataset=dataset_all,
+        output_split_filename=os.path.join(paths["data_misc_dir"], paths["data_split_filename"]),
+        id="data.patientID",
+        keys_to_balance=["data.gt.classification"],
+        nfolds=train["num_folds"],
+        workers=train["num_workers"],
+    )
+
+    train_sample_ids = []
+    for fold in train["train_folds"]:
+        train_sample_ids += folds[fold]
+    validation_sample_ids = []
+    for fold in train["validation_folds"]:
+        validation_sample_ids += folds[fold]
+
+    train_dataset = PICAI.dataset(
+        paths=paths,
+        cfg=train,
+        reset_cache=False,
+        sample_ids=train_sample_ids,
+        train=True,
+    )
+
+    validation_dataset = PICAI.dataset(paths=paths, cfg=train, reset_cache=False, sample_ids=validation_sample_ids)
+
+    ## Create sampler
+    lgr.info("- Create sampler:")
+    sampler = BatchSamplerDefault(
+        dataset=train_dataset,
+        balanced_class_name="data.gt.classification",  # gt_label, TODO - make diff label for balance-sampler
+        num_balanced_classes=2,
+        batch_size=train["batch_size"],
+        mode="approx",
+        workers=train["num_workers"],
+        balanced_class_weights=None,
+    )
+
+    lgr.info("- Create sampler: Done")
+
+    ## Create dataloader
+    train_dataloader = DataLoader(
+        dataset=train_dataset,
+        shuffle=False,
+        drop_last=False,
+        batch_sampler=sampler,
+        collate_fn=CollateDefault(
+            special_handlers_keys={"data.input.img": CollateDefault.pad_all_tensors_to_same_size},
+            raise_error_key_missing=False,
+        ),
+        num_workers=train["num_workers"],
+    )
+    lgr.info("Train Data: Done", {"attrs": "bold"})
+
+    #### Validation data
+    lgr.info("Validation Data:", {"attrs": "bold"})
+
+    ## Create dataloader
+    validation_dataloader = DataLoader(
+        dataset=validation_dataset,
+        shuffle=False,
+        drop_last=False,
+        batch_sampler=None,
+        batch_size=train["batch_size"],
+        num_workers=train["num_workers"],
+        collate_fn=CollateDefault(),
+    )
+    lgr.info("Validation Data: Done", {"attrs": "bold"})
+
+    # ====================================================================================
+    #  Loss and metrics
+    # ====================================================================================
+    # TODO - add a classification loss - add head to the bottom of the unet
+    losses = {}
+    losses["segmentation"] = LossDefault(
+        pred="model.seg",
+        target="data.gt.seg",
+        callable=monai.losses.DiceLoss(to_onehot_y=True, softmax=False),
```

Comment:
do we have diceloss in fuse?

Reply:
we have something, but it's not suited.
will create our fixed version after finishing current code PR fix